cyhy/mailer/cli.py

```diff
@@ -312,23 +315,30 @@ def send_message(ses_client, message, counter=None):
     anything other than 200.
 
     """
-    # "Are you silly?  I'm still gonna send it!"
-    #   -- Larry Enticer
-    response = ses_client.send_raw_email(RawMessage={"Data": message.as_string()})
-
-    # Check for errors
-    status_code = response["ResponseMetadata"]["HTTPStatusCode"]
-    if status_code != 200:
-        logging.error(f"Unable to send message.  Response from boto3 is: {response}")
-        raise UnableToSendError(response)
+    if not dry_run:
+        # "Are you silly?  I'm still gonna send it!"
+        #   -- Larry Enticer
+        response = ses_client.send_raw_email(RawMessage={"Data": message.as_string()})
+
+        # Check for errors
+        status_code = response["ResponseMetadata"]["HTTPStatusCode"]
+        if status_code != 200:
+            logging.error(
+                f"Unable to send message.  Response from boto3 is: {response}"
+            )
+            raise UnableToSendError(response)
+    else:
+        logging.debug("NOT sending message")
```

Comment:
It would be nice if this debug message mentioned that the message is not being sent because of `--dry-run`.  How about something like this?
```suggestion
        logging.debug("NOT sending message (dry run)")
```